docbook/reference/en/en-US/modules/Installation_Configuration.xml

```diff
@@ -278,8 +277,8 @@
         </para>
         <para>
             Since we're not using a &lt;servlet-mapping&gt; element, we must define an
-            <classname>Application</classname> class (see <xref linkend="javax.ws.rs.core.Application"/>) that is annotated with
-            the <classname>@ApplicationPath</classname> annotation.  If you return any empty set for classes and singletons,
+            <classname>javax.ws.rs.core.Application</classname> class (see <xref linkend="javax.ws.rs.core.Application"/>) that is annotated with

```

Comment:
I'm not a native English speaker, but should it be "a javax.ws.rs.core.Application class" instead of "an javax.ws.rs.core.Application class" ?